FreshRSS/FreshRSS
# Improved: navigation menu structure

similar to #4891

The left menu in subscription management + configuration has now a better list structure that represents the true semantic tree: "headlines" and its "sub-trees".

Changes proposed in this pull request:

- (S)CSS


How to test the feature manually:

1. check it with al themes
2. see the left hand navigation bar: subscription management
3. see the left hand navigation bar: configuration 

Pull request checklist:
